src/components/Main/Teams/TeamList.tsx

```diff
@@ -1,11 +1,10 @@
-import React from 'react';
+import React, { useState } from 'react';
```

Comment:
`useState` isn't currently used in `TeamList.tsx`

Reply:
good catch. I wanted to make a hook to not allow unused imports to go through pull requests.. but too much work haha